remotes/ref-and-definitions.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$id": "ref-and-definitions.json",
+    "definitions": {
+        "inner": {
+            "properties": {
+                "bar": { "type": "string" }
+            }
+        }
+    },
+    "$ref": "#/definitions/inner"
```

Comment:
I thought this was only true for verification keywords (e.g. `properties`), not for stuff like `definitions`. These test definitely pass on the client I'm using (JsonSchema.Net), meaning that `definitions` is found.

Reply:
It does work in many implementations, but not all. Whether an implementation supports it accidentally or on purpose, the ambiguity is reason enough to avoid it.

Reply:
Yeah, I looked and the updated standard indeed writes "all other keywords". I changed it.